test/src/sender/main.cpp

```diff
@@ -78,63 +95,83 @@ using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
+        cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen,
+	    TIME, const char* >("generator_con" ,std::move(i_input_data_control));
 
-string input_data_ack = "test/data/sender/sender_input_test_ack_In.txt";
-const char * i_input_data_ack = input_data_ack.c_str();
+    string input_data_ack = "test/data/sender/sender_input_test_ack_In.txt";
+    const char * i_input_data_ack = input_data_ack.c_str();
 
-std::shared_ptr<cadmium::dynamic::modeling::model> generator_ack = cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen, TIME, const char* >("generator_ack" , std::move(i_input_data_ack));
+    std::shared_ptr<cadmium::dynamic::modeling::model> generator_ack =
+        cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen,
+		TIME, const char* >("generator_ack" ,std::move(i_input_data_ack));
 
 
 /********************************************/
 /****** SENDER *******************/
 /********************************************/
 
-std::shared_ptr<cadmium::dynamic::modeling::model> sender1 = cadmium::dynamic::translate::make_dynamic_atomic_model<Sender, TIME>("sender1");
+    std::shared_ptr<cadmium::dynamic::modeling::model> sender1 =
+        cadmium::dynamic::translate::make_dynamic_atomic_model<Sender,
+		TIME>("sender1");
 
 
 /************************/
 /*******TOP MODEL********/
 /************************/
-cadmium::dynamic::modeling::Ports iports_TOP = {};
-cadmium::dynamic::modeling::Ports oports_TOP = {typeid(outp_data),typeid(outp_pack),typeid(outp_ack)};
-cadmium::dynamic::modeling::Models submodels_TOP = {generator_con, generator_ack, sender1};
-cadmium::dynamic::modeling::EICs eics_TOP = {};
-cadmium::dynamic::modeling::EOCs eocs_TOP = {
-  cadmium::dynamic::translate::make_EOC<Sender_defs::packetSentOut,outp_pack>("sender1"),
-cadmium::dynamic::translate::make_EOC<Sender_defs::ackReceivedOut,outp_ack>("sender1"),
-cadmium::dynamic::translate::make_EOC<Sender_defs::dataOut,outp_data>("sender1")
-};
-cadmium::dynamic::modeling::ICs ics_TOP = {
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,Sender_defs::controlIn>("generator_con","sender1"),
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,Sender_defs::ackIn>("generator_ack","sender1")
-};
-std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP = std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
- "TOP",
- submodels_TOP,
- iports_TOP,
- oports_TOP,
- eics_TOP,
- eocs_TOP,
- ics_TOP
-  );
+    cadmium::dynamic::modeling::Ports iports_TOP = {};
+    cadmium::dynamic::modeling::Ports oports_TOP = {
+        typeid(output_data),typeid(output_pack),typeid(output_ack)
+    };
+    cadmium::dynamic::modeling::Models submodels_TOP = {
+        generator_con, generator_ack, sender1
+    };
+    cadmium::dynamic::modeling::EICs eics_TOP = {};
+    cadmium::dynamic::modeling::EOCs eocs_TOP = {
+        cadmium::dynamic::translate::make_EOC<Sender_defs::packetSentOut,
```

Comment:
consider changing the structure names to sender_defs , packet_sent_output throughout

Reply:
changed after review

Reply:
verified